core/common/src/main/java/alluxio/grpc/GrpcUtils.java

```diff
@@ -718,6 +718,15 @@ public static boolean contains(Scope source, Scope target) {
     return (source.getNumber() | target.getNumber()) == source.getNumber();
   }
 
+  /**
+   * @param source source enum
+   * @param target target enum
+   * @return true if source enum is contained within the target scope
+   */
+  public static boolean containsInScope(Scope source, Scope target) {
```

Comment:
is this `contains(source, target) -> contains(target, source)`?

Reply:
Not really, i think so at first glance